docs/reference/user-management.md

```diff
@@ -0,0 +1,118 @@
+:::
+
+### Step two: in-app setup
+
+When you set up user management for the first time, you create an owner account.
+
+1. Open n8n. The app displays a signup screen.
+2. Enter your details. Your password must be at least eight characters, including at least one number and one capital letter.
+3. Click **Next**. n8n logs you in with your new owner account.
+
+### Step three: invite users
+
+You can now invite other people to your n8n instance. 
+
+1. Sign in with your owner account.
+2. Click your user icon > **Settings**. n8n opens your **Personal settings** page.
+3. Click **Users** to go to the **Users** page.
+4. Click **Invite**.
+5. Enter the new user's email address.
+6. Click **Invite user**. n8n sends an email with a link for the new user to join.
+
+## Manage users
+
+The **Users** page shows all users, including ones with pending invitations. You can:
+
+* Delete a user: 
+  1. Click the menu icon by the user you want to delete.
+  2. Confirm you want to delete them.
+  3. If they are an active user, choose whether to copy their workflow data and credentials to a new user, or permanently delete their workflows and credentials.
+* Resend an invitation to a pending user: click the menu icon by the user, then click **Resend invite**.
+
+
+## Skipping or disabling user management
+
+You don't have to use n8n's user management feature. You can:
+
+* Leave it enabled, but choose to skip the setup step. You can use n8n as normal. If you want to set up user management later, go to **Settings** > **Users**.
+* Disable the feature completely using the `N8N_USER_MANAGEMENT_DISABLED` environment variable. Setting this environment variable to `true` completely hides the feature in your n8n instance. You can't use this setting if you have already set up an owner account.
+
+## Best practices
+
+This sections contains advice on best practices relating to user management in n8n.
+
+* We recommend that owners create a member-level account for themselves. Owners can see all workflows, but there is no way to see who created a particular workflow, so there is a risk of overriding other people's work if you build and edit workflows as an owner.
+* Users must be careful not to edit the same workflow simultaneously. It is possible to do it, but the users will overwrite each other's changes.
+* Webhook paths must be unique across the entire instance. This means each webhook path must be unique for all workflows and all users. By default, n8n generates a long random value for the webhook path, but users can edit this to their own custom path. If two users set the same path value:
+    * The path works for the first workflow that is run or activated.
+    * Other workflows will error if they try to run with the same path.
+* If you run n8n behind a reverse proxy, set the following environment variables so that emails are generated with the correct URL:
+  * `N8N_HOST`
+  * `N8N_PORT`
+  * `N8N_PROTOCOL`
```

Comment:
This should be updated to reflect the new `N8N_EDITOR_BASE_URL` environment variable. 
(cf. [commit](https://github.com/n8n-io/n8n/pull/2967/commits/adc0f32c52784e77a42262ffcb39c776fda7bdb6))

Reply:
Added